src/app/App.tsx

```diff
@@ -55,7 +55,6 @@ class App extends React.Component<AppProps, AppState> {
         <div className="settings">
           <StyleLoader
             parsers={[
-              OlStyleParser,
```

Comment:
What is the exact reason, why you remove the OL Style Parser?

Reply:
I removed it because there are actually no Ol Style Files that could be read. And as i didn't want the app to become to complex i decided to just remove the OlStyle as Upload Input.